packages/rocketchat-livechat/server/lib/Livechat.js

```diff
@@ -311,9 +311,31 @@ RocketChat.Livechat = {
 		});
 	},
 
-	savePageHistory(token, pageInfo) {
+	savePageHistory(token, roomId, pageInfo) {
```

Comment:
you don't need to receive `roomId` here.. we can rely on `RocketChat.models.Rooms.findOneOpenByVisitorToken` to get visitor's current room.

Reply:
@sampaiodiego, the `RocketChat.models.Rooms.findOneOpenByVisitorToken` has a `roomId` parameter, due to this I changed the `savePageHistory` to get this parameter.
How can I call the `RocketChat.models.Rooms.findOneOpenByVisitorToken` method if I don't pass the `roomId` parameter?
I'm sorry if I'm making a mistake but I think this parameter is necessary.

Reply:
you're correct @renatobecker .. I didn't notice `RocketChat.models.Rooms.findOneOpenByVisitorToken ` has a `roomId` parameter as well. 😉 
I'll give this PR a whole review to see how we could still save visitor's navigation history without a room.